### DISCUSSION TITLE
How to set the font

### DISCUSSION THREAD

**Discussion by moon989** [slint-ui/slint#996] [Q&A]
i got following error when run demo.
 **panicked at 'there must be a sans-serif font face registered**'. 
 I am newer for this , i wanna know  How to registe  the font?

**Comment by tronical** [comment]
This panic suggests that we've failed in detecting systems fonts on your machine. We may need to improve your detection. What operating system are you using?

**Reply by moon989** [reply]
Thanks for repley.    I use Deepin linux system which is chinese OpenSource Linux System.  Can i registe a special font  at  first before running.

**Reply by tronical** [reply]
Thanks, that is a system I don't have experience with. When you run `fc-match sans-serif` in a terminal, what is the output that is printed? (similarly, what is the output if you run just `fc-match` (without any arguments))

**Reply by moon989** [reply]
run  _fc-match_    got :  

>  wqy-microhei.ttc: "文泉驿微米黑" "Regular"

**Reply by tronical** [reply]
Thanks! Is that the same output for `fc-match sans serif` and `fc-match` without any arguments?

Hm, I might just have to install deepin linux in a vm.
